appveyor.yml

```diff
@@ -34,14 +34,14 @@ build:
   publish_nuget: true
   verbosity: minimal
   publish_nuget_symbols: true
+  use_snupkg_format: true

```

Comment:
@Yuri05 This was missing. We create symbol packages in the old format. We might as well use the new format supported by nuget

Reply:
interesting..